examples/client_simple/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/teeworlds-go/protocol/protocol7"
+	"github.com/teeworlds-go/protocol/teeworlds7"
+)
+
+func main() {
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
+	defer cancel()
+
+	client := teeworlds7.NewClient()
+	client.Name = "nameless tee"
+
+	client.OnPacket(func(packet *protocol7.Packet) bool {
+		fmt.Printf("received packet w/ %d messages\n", len(packet.Messages))
```

Comment:
This seems a bit verbose for a "client_simple" given that there is already a "client_verbose"

Also I personally am not a big fan of "w/" ._.

Reply:
resolved

Reply:
reverted message, removed newline & punctuation.